content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,305 @@
+  - Choose a different Datadog site than your primary (for example if you're on `US1`, choose `EU` or `US5`)
+  - Follow the prompts to create an account
+
+
+All Datadog sites are geographically separated. Reference the [Datadog Site List](https://docs.datadoghq.com/getting_started/site#access-the-datadog-site) for options. 
+
+If you are also sending telemetry to Datadog using cloud provider integrations, you must add your cloud provider accounts in the DDR org. Datadog does not use cloud providers to receive telemetry data while the DDR site is passive (not in failover).
+
+**To share the DDR org information with Datadog:**
+
+- Email your DDR org name to your [Customer Success Manager](mailto:success@datadoghq.com)
+- Your Customer Success Manager sets this new org as your **DDR failover organization**
+
+
+**Note:** Although this organization appears in your Datadog billing hierarchy, all usage and cost associated is _not_  billed during the Preview period.
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Retrieve the public IDs and link your DDR and primary orgs " level="h5" %}}
+
+After the Datadog team has set your DDR org, use the cURL commands from the Datadog [public API endpoint][8] to retrieve the public IDs of the primary and DDR org. 
+
+
+To link your DDR and primary orgs, run these commands replacing the placeholders for their values:
+
+<div class="alert alert-warning"> For security reasons, Datadog is unable to link the orgs on your behalf. </div>
+
+
+```shell
+export PRIMARY_DD_API_KEY=<PRIMARY_ORG_API_KEY>
+export PRIMARY_DD_APP_KEY=<PRIMARY_ORG_APP_KEY>
+export PRIMARY_DD_API_URL=<PRIMARY_ORG_API_SITE>
+
+export DDR_ORG_ID=<DDR_ORG_PUBLIC_ID>
+export PRIMARY_ORG_ID=<PRIMARY_ORG_PUBLIC_ID>
+export USER_EMAIL=<USER_EMAIL>
+export CONNECTION='{"data":{"id":"'${PRIMARY_ORG_ID}'","type":"hamr_org_connections","attributes":{"TargetOrgUuid":"'${DDR_ORG_ID}'","HamrStatus":1,"ModifiedBy":"'${USER_EMAIL}'", "IsPrimary":true}}}'
+
+curl -v -H "Content-Type: application/json" -H 
+"dd-api-key:${PRIMARY_DD_API_KEY}" -H 
+"dd-application-key:${PRIMARY_DD_APP_KEY}" --data "${CONNECTION}" --request POST ${PRIMARY_DD_API_URL}/api/v2/hamr
+```
+{{% /collapse-content %}}
+
+<br>
+
+<!-- ------------------------------- -->
+
+
+### 2. Set up access, integrations, syncing, and agents
+
+{{% collapse-content title="Create your Datadog API and App key for syncing" level="h5" %}}
+In DDR Datadog org, create a set of `API key` **and** `App key`. These are useful for copying dashboards and monitors between Datadog sites. 
+
+<div class="alert alert-info">
+Datadog can help copy the API key signatures for your Agents to the DDR back-up account. Contact your <a href="mailto:success@datadoghq.com">Customer Success Manager</a> for any questions regarding this.
+</div>
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Configure Single Sign On for the Datadog App" level="h5" %}}
```

Comment:
```suggestion
{{% collapse-content title="Configure Single Sign On for the DDR org" level="h5" %}}
```
The wording made me think you were talking about the mobile app at first. But I think this is important on the org level, rather than on the desktop app level?